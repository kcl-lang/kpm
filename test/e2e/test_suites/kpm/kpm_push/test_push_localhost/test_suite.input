kpm push oci://localhost:5001/test/helloworld