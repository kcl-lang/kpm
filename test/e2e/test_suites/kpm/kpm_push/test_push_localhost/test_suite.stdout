package 'helloworld' will be pushed
pushed [registry] localhost:5001/test/helloworld